crates/viewer/re_space_view_dataframe/src/dataframe_ui.rs

```diff
@@ -393,7 +371,7 @@ impl<'a> egui_table::TableDelegate for DataframeTableDelegate<'a> {
                     column.data_ui(
                         self.ctx,
                         ui,
-                        row_id,
+                        RowId::ZERO,
```

Comment:
Will that be an issue @abey79 ? (I tried to test this myself, but there seems to be an unrelated issue on `main` at the moment, see #tech).

Reply:
This `row_id` ultimately is routed to `ComponentUiRegistry::ui_raw`, which takes an `Option<RowId>`. If we no longer have a row id there, the correct patch is to remove it entirely and pass `None` in the `ui_raw` call site  (`fn data_ui()`  in `display_record_batch.rs`).

As to the effect of doing that, I can't even track where its actually used --> discussion on slack.

Reply:
If this blocks you, remove that `RowId` thing entirely and we'll copy later if that proves to be an actual issue. Gut feeling: some hovertooltip might end up being incorrect in edge cases.

Reply:
As discussed, the effect of this removal will be that the table cell will no longer preview tiny thumbnail of images. This actually is a good thing because this has performance issues anyways. Some more background in:
- #7617.